int16[] age_list
string[] sex_list
string[] up_color_list
string[] down_color_list
string[] glasstf_list